pinot-common/src/main/java/org/apache/pinot/common/metadata/segment/SegmentZKMetadata.java

```diff
@@ -323,6 +324,15 @@ public void setEndOffset(String endOffset) {
     setValue(Segment.Realtime.END_OFFSET, endOffset);
   }
 
+  public StreamContinuationMode getContinuationMode() {
+    return _znRecord.getEnumField(Segment.Realtime.CONTINUATION_MODE, StreamContinuationMode.class,
+        StreamContinuationMode.RESUME);
```

Comment:
Do you see we might add another mode in the future? I feel a `boolean` field of whether it is the first segment of a streaming partition is good enough.